packages/api/src/schema/schema.yaml

```diff
@@ -2567,90 +2518,148 @@ components:
 
 paths:
   /stream:
-    tags:
-      - stream
     post:
-      description: Creates a stream
+      summary: Create a new stream
+      description: This endpoint allows users to create a new stream resource.
       requestBody:
+        description: Stream resource to be created
         required: true
         content:
           application/json:
             schema:
               type: object
               $ref: "#/components/schemas/stream"
-
       responses:
-        200:
+        "200":
           description: Success
           content:
             application/json:
               schema:
                 type: array
                 items:
                   $ref: "#/components/schemas/stream"
+        "401":
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+        "500":
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
```

Comment:
Could we have like just 1 entry like "error response"? We don't return only 401 or 500 from these APIs, so I think we can both simplify the docs as well as be more extensive. Could specify the status code as "4xx or 5xx" or "> 400" here, or something like that. That should also replace that "default" response listed in these. WDYT?

Reply:
Sounds great!